### DISCUSSION TITLE
Accidentally overwrote settings from another PC

### DISCUSSION THREAD

**Discussion by codedabbling** [runelite/runelite#15674] [Q&A] [answered]
Hi,

I accidentally overwrote all of my settings when signing in to a new pc. I was hoping there was a way to revert back to my old settings... I had so much changed.

**Comment by Enriath** [comment, marked as answer]
The import system creates a backup of your old signed-in settings before importing the signed-out ones, in `.runelite/profiles/<email>`
You can copy this file into `.runelite`, remove the existing `settings.properties` file, rename the backup to remove the timestamp, and import it like normal.
